scripts/tms

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Get the absolute path to the library
+LIB_PATH=$(python -c "
+import torch_memory_saver, importlib.machinery, pathlib, os
+package_dir = pathlib.Path(torch_memory_saver.__file__).parent
+ext_suffix = importlib.machinery.EXTENSION_SUFFIXES[0]
+lib_name = f'torch_memory_saver_cpp{ext_suffix}'
+print(str(package_dir / lib_name))
+")
+
+# Detect if script is being sourced
+(return 0 2>/dev/null) && SOURCED=1 || SOURCED=0
```

Comment:
nit: curious whether this will accurately detect whether it is being sourced, or will there false positive / negative?